x/twap/listeners.go

```diff
@@ -24,8 +26,12 @@ type epochhook struct {
 	k Keeper
 }
 
+func (k Keeper) EpochHooks() epochtypes.EpochHooks {
```

Comment:
Good catch. Do we also need to set it here?
https://github.com/osmosis-labs/osmosis/blob/b1c36e8f0a75b434e3354d355642f425b6c7e3cc/app/keepers/keepers.go#L489-L497

Reply:
ah good catch!